src/pages/About/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { SignIn } from "../../components/SignIn";
+import { useHistory } from 'react-router'
+import { useUserState } from "../../store/user/useUserState";
+import Button from '@material-ui/core/Button';
+import "./About.css";
+import { Paper } from "@material-ui/core";
+
+export const About = () => {
+  const { userData } = useUserState();
+  
+  let history = useHistory();
+  const linkHandler = (page: string) => {
+    history.push(page);
+  };
```

Comment:
This function its just redirect to a specific page so, I think you it doesn't need to receive a path and add the `/user/myGardens directly`, unless you are going use it to redirect to different pages

Also `let history = useHistory();` => `const`

Reply:
Understood!